flink/interpreter/src/test/java/org/apache/zeppelin/flink/FlinkStreamSqlInterpreterTest.java

```diff
@@ -253,6 +254,67 @@ public void testResumeStreamSqlFromSavePoint() throws IOException, InterpreterEx
             resultMessages.get(0).getData().contains("url\tpv\n"));
   }
 
+  @Test
+  public void testResumeStreamSqlFromExistSavePointPath() throws IOException, InterpreterException, InterruptedException, TimeoutException {
```

Comment:
Could you add another test that resume from invalid savepointPath ?

Reply:
I will finish them soon